### DISCUSSION TITLE
Heading-to-tag transformation

### DISCUSSION THREAD

**Discussion by exanrauzer** [Zettlr/Zettlr#4445] [Q&A] [answered]
Hey there!

I have a question regarding the default behavior of transforming headings into tags in beta5. I'm not quite sure if this is a bug or if it's intended to work this way. Whether this behavior is by design or if there might be an issue lurking beneath the surface?

Furthermore, keywords composed of two words that are treated as tags are suggested after "#" and a space.

Many thanks for beta5!

**Comment by nathanlesage** [comment, marked as answer]
So I did in fact make the Tag autocomplete more aggressive so that it shows you tags even without having to type one letter. That annoyed me a little bit because somehow I want to see what tags I have before I begin to type one. The reason is that headings must have a space in between anyways. Therefore the second behavior you describe is a bug -- I'll need to remove multi-word tags from that autocomplete since it'll only work without spaces anyway.

**Reply by exanrauzer** [reply]
I'm afraid I didn't express myself accurately. <s>I've noticed that some of my headings have been transformed into tags (I thought that all of them, but clearly not). The issue is that I find #some_heading as a tag, even though it's written as ### some_heading in the document, and there is no #some_heading tag anywhere across the documents.</s>

Just to clarify, I have no problem with the aggressive suggestion retrieval of tasks immediately after typing "#". It's actually pretty cool. And thanks for explaining the "second behavior" - I'll create an issue for that if it's alright with you."

**Reply by exanrauzer** [reply]
Apologies, I actually included a few words in the keywords section of the YAML that I missed earlier! I'm sorry for the confusion caused!
